### ISSUE TITLE
more remote storage options

### ISSUE THREAD

**Issue by WuTheFWasThat** [WuTheFWasThat/vimflowy#236]

**Comment by LeapingDragon** [comment]
[remoteStorage](https://remotestorage.io/) looks like a good fit - allows hosting user data on any remoteStorage provider, Google Drive or Dropbox. [Laverna](https://laverna.cc/) and some [other apps](https://wiki.remotestorage.io/Apps) use it as well

**Comment by WuTheFWasThat** [comment]
yeah - it was mentioned before in #124 and does look like it could work well!  I've not had the time/energy to try it yet, but it may be relatively simple (for those curious, you can see the class FirebaseStore in datastore.ts for reference)

**Comment by davekaufman** [comment]
webdav support?  would allow for easy storage into owncloud...

**Comment by WuTheFWasThat** [comment]
I've added a server that can be used which stores data in sqlite
